bin/i18n/crowdin/config/codeorg-markdown-testing_crowdin.yml

```diff
@@ -1,15 +1,16 @@
-"project_identifier" : "codeorg-markdown-testing"
```

Comment:
`project_identifier` is a remanent from the old Crowdin API V1. We have a variable that associates `project_identifier` with `project_id`. Instead of that workaround, we are reading the project_id from the file now.